table/block_based/block_based_table_builder.h

```diff
@@ -170,8 +170,8 @@ class BlockBasedTableBuilder : public TableBuilder {
   void CompressAndVerifyBlock(
       const Slice& raw_block_contents, bool is_data_block,
       CompressionContext& compression_ctx, UncompressionContext* verify_ctx,
-      std::string& compressed_output, Slice& result_block_contents,
-      CompressionType& result_compression_type, Status& out_status);
+      std::string* compressed_output, Slice* result_block_contents,
```

Comment:
What's the reason of changing the reference to pointers?

Reply:
Hmm... Now it feels irrelevant to this diff. Initially it was relative as I changed out_status that passed in.

Anyway, Google C++ Style bans non-constant reference as function argument. Output arguments need to be pointers.